src/main/java/com/epam/jdbc/service/serviceimpl/UserServiceImpl.java

```diff
@@ -18,6 +18,7 @@ public class UserServiceImpl implements UserService<User> {
 
     @Override
     public User findUserByEmail(String email) {
+        logger.info("Find user by {} email", email);
```

Comment:
Please remove one of logs, because there are two same logs in this method.

Reply:
done